pytorchfi/neuron_error_models.py

```diff
@@ -13,6 +13,10 @@ def random_batch_element(pfi):
     return random.randint(0, pfi.get_total_batches() - 1)
 
 
+def random_value(min_val=-1, max_val=1):
```

Comment:
I think duplicating this function (appears in `neuron_error_models` and `weight_error_models`) isn't good practice – I think that since its not specific to any we can create a `util` file.